mri/optimizers/utils/cost.py

```diff
@@ -22,7 +22,7 @@ class DualGapCost(costObj):
     """ Define the dual-gap cost function.
     """
     def __init__(self, linear_op, initial_cost=1e6, tolerance=1e-4,
-                 cost_interval=1, test_range=4, verbose=False,
+                 cost_interval=None, test_range=4, verbose=False,
```

Comment:
I feel like this is unrelated

Reply:
This is just to speed up reconstruction as currently, we calculate the cost at every iteration.

It is related to this PR, only in the sense that we are changing cost related stuff here. 
It is just a single line change.

Reply:
Makes sense.